VaticanBAV/et/BAVet132.xml

```diff
@@ -0,0 +1,424 @@
+                                <item xml:id="q15" n="14">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="127" to="136"/>
+                                </item>
+                                <item xml:id="q16" n="15">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="137" to="146"/>
+                                </item>
+                                <item xml:id="q17" n="16">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="147" to="156"/>
+                                </item>
+              <item xml:id="q18" n="17">
+                <dim unit="leaf">8</dim>
+                <locus from="157" to="164"/>
+              </item>
+              <item xml:id="q19" n="18">
+                <dim unit="leaf">5</dim>
+                <locus from="165" to="169"/>
+              </item>
+             
+                              </list>-->
+                            </collation>
+                          </supportDesc>
+                        
+                        <layoutDesc>
+                          <layout columns="1" writtenLines="28">
+                            <locus from="3r" to="111r"/>
+                          </layout>
+                          <layout columns="2" writtenLines="28">
+                            <locus from="111v" to="126r"/>
+                          </layout>
+                        </layoutDesc>
+                        </objectDesc>
+                      
+                        <handDesc>
+                          <handNote script="Ethiopic" xml:id="h1">
+                            <desc>The old form of the numeral <foreign xml:lang="gez">፲</foreign> 
+                            with a circle attached to the body with a line, is visible on <locus from="76v" to="77v"/>,
+                              from which it seems that the manuscript was copied from an older exemplar.
+                            </desc>
+                            <seg type="ink">Black, red.</seg>
+                            <seg type="rubrication">Titles.</seg>
+                            <date notBefore="1800" notAfter="1899"/>
+                          </handNote>
+                        </handDesc>
+                      
+                      <decoDesc>
+                        <decoNote xml:id="d1" type="band">
+                          <desc>Ornamental bands in black and red are frequently present at the beginning of
+                          textual units and after almost every ten psalms.</desc>
+                        </decoNote>
+                        
+                        <decoNote xml:id="d2" type="printedImage" cert="low">
+                          <desc>At the beginning of the Psalter, a strip is covered by a piece of paper showing flowers and pink leaves
+                            on a blue page.</desc>
```

Comment:
turquoise?

Reply:
is by any chance fascia the term they otherwise use for band/harag?

Reply:
No, I don't think so :(

Reply:
was just trying to make sense, cannot visualize what they meant)) very weird phrase, but the translation seems correct. we have to wait till they have digitized ))